examples/example-create-01.php

```diff
@@ -48,8 +41,8 @@
 namespace CFPropertyList;
 
 // just in case...
-error_reporting( E_ALL );
-ini_set( 'display_errors', 'on' );
+error_reporting(E_ALL);
```

Comment:
Is it really good to have this?.... is not better no have it as an argument or ENV set?

Reply:
These are examples to show how to use the library. They must be as simple as possible, straighforward. For me this is fine.